cmd/cored/main.go

```diff
@@ -116,6 +117,17 @@ func init() {
 }
 
 func main() {
+	v := flag.Bool("version", false, "prints cored version")
+	flag.Parse()
+
+	if *v {
+		fmt.Printf("cored version: %s production=%t\n", config.Version, config.Production)
```

Comment:
my rationale:

- less stutter (the command already says `-version`)
- closer to the [GNU style](https://www.gnu.org/prep/standards/standards.html#g_t_002d_002dversion) (for whatever that's worth)
- looks nicer to my eye